cmd/skuba/cluster/init.go

```diff
@@ -32,6 +35,22 @@ type initOptions struct {
 	StrictCapDefaults bool
 }
 
+// Checks if clustername provided is correct
+func isValidClusterName(clustername string) bool {
+	// NOT a letter, a number, underscore/dash
+	reg, err := regexp.Compile("[^a-zA-Z0-9_-]+")
```

Comment:
is a `.` a character we also want to allow?

Reply:
you tell me! :)

Reply:
Let's block it for the moment and let's see what happens later.